core/types/imports.go

```diff
@@ -23,11 +23,14 @@ type (
 	BlobTxSidecar     = ethtypes.BlobTxSidecar
 	Signer            = ethtypes.Signer
 	HomesteadSigner   = ethtypes.HomesteadSigner
-
-	Transaction  = ethtypes.Transaction
-	Transactions = ethtypes.Transactions
-	TxByNonce    = ethtypes.TxByNonce
-	TxData       = ethtypes.TxData
+	StateAccount      = ethtypes.StateAccount
+	SlimAccount       = ethtypes.SlimAccount
+	Header            = ethtypes.Header
+	BlockNonce        = ethtypes.BlockNonce
+	Transaction       = ethtypes.Transaction
+	Transactions      = ethtypes.Transactions
+	TxByNonce         = ethtypes.TxByNonce
+	TxData            = ethtypes.TxData
```

Comment:
We might want to sort these (and existing ones too) alphabetically to have a consistent order with subnet-evm as well.

Reply:
Done in #800